clang/include/clang/Interpreter/Interpreter.h

```diff
@@ -49,22 +49,25 @@ class IncrementalCompilerBuilder {
   }
 
   // General C++
-  llvm::Expected<std::unique_ptr<CompilerInstance>> CreateCpp();
+  llvm::Expected<std::unique_ptr<CompilerInstance>> CreateCpp(llvm::Triple TT);
```

Comment:
I was wondering since we are using a builder class would it make sense to have a separate API to set the triple if it was different from the default?

Reply:
Yes, makes sense